app/scripts/modules/ecs/src/serverGroup/configure/wizard/networking/networkingSelector.component.html

```diff
@@ -94,62 +94,4 @@
       </div>
     </div>
   </div>
-
-  <div class="col-md-12" ng-if="$ctrl.command.viewState.dirty.targetGroup">
-    <div class="alert alert-warning">
-      <p>
-        <i class="fa fa-exclamation-triangle"></i>
-        The following target groups could not be found in the selected account/region/VPC and were removed:
```

Comment:
Are we losing this logic? What happens when I select a different vpc higher up in the UI?

Reply:
I don't think this logic actually works. Dirty.TargetGroups is never being set for the ECS provider (it is though for the Amazon provider)

Reply:
@clareliguori would you have any objection if I implemented this in a separate PR?

Reply:
No, provided that it doesn't actually work today, so we're not losing anything